modules/n1ql/pages/n1ql-language-reference/transactions.adoc

```diff
@@ -309,9 +324,9 @@ include::example$transactions/results.jsonc[tags=!ellipsis]
 ----
 
 <.> Beginning a transaction returns a transaction ID.
-<.> Before setting the second savepoint, the booking document has user `"0"`, name `"Keon Hoppe"`.
-<.> After setting the second savepoint and performing an update, the booking document has user `"1"`, name `"Rigoberto Bernier"`.
-<.> After rolling back to the second savepoint, the booking document again has user `"0"`, name `"Keon Hoppe"`.
+<.> Before setting the second savepoint, the booking document has user `"0"`, name `"Maurice Moss"`.
```

Comment:
Updating these callouts feels a bit onerous doesn't it?
(In fact, we've deprecated callouts in the style guide, might want to consider deleting them entirely)

Reply:
Deleting the callouts is outside the scope of this ticket, I would say?

Reply:
I'll put in on the improvements list

Reply:
[AV-54655](https://couchbasecloud.atlassian.net/browse/AV-54655)